universal-application-tool-0.0.1/app/views/admin/programs/ProgramNewOneView.java

```diff
@@ -21,37 +21,25 @@ public ProgramNewOneView(AdminLayout layout) {
   }
 
   public Content render(Request request) {
-    ProgramForm blankForm = new ProgramForm();
-    return layout.render(
-        body(
-            renderHeader("New program"),
-            div(
-                ProgramFormBuilder.buildProgramForm(
-                        blankForm.getAdminName(),
-                        blankForm.getAdminDescription(),
-                        blankForm.getLocalizedDisplayName(),
-                        blankForm.getLocalizedDisplayDescription(),
-                        false)
-                    .with(makeCsrfTokenInputTag(request))
-                    .withAction(controllers.admin.routes.AdminProgramController.create().url()))));
+    return render(request, new ProgramForm(), "");
   }
 
   public Content render(Request request, ProgramForm programForm, String message) {
-    ContainerTag bodyContent =
-        body(
-            renderHeader("New program"),
-            div(
-                ProgramFormBuilder.buildProgramForm(
-                        programForm.getAdminName(),
-                        programForm.getAdminDescription(),
-                        programForm.getLocalizedDisplayName(),
-                        programForm.getLocalizedDisplayDescription(),
-                        false)
-                    .with(makeCsrfTokenInputTag(request))
-                    .withAction(controllers.admin.routes.AdminProgramController.create().url())));
-    if (message.length() > 0) {
-      bodyContent.with(ToastMessage.error(message).setDismissible(false).getContainerTag());
+    String title = "New program";
```

Comment:
discussion topic that doesn't need to be solved in this PR (non-blocking): We should eventually standardize on either using "Sentence case" or "Title Case" for our page titles.

"Sentence case" (i.e., only capitalizing the first letter of the first word rather than capitalizing the first letter of every word) is what we use on my other team, and I personally think it looks nice. But most important is definitely just consistency across CiviForm.